FastSurferCNN/README.md

```diff
@@ -33,19 +38,21 @@ The *FastSurferCNN* directory contains all the source code and modules needed to
                     If this fails, or you actively overwrote the check with setting "--run_viewagg_on cpu", view agg is run on the cpu. 
                     Equivalently, if you define "--run_viewagg_on gpu", view agg will be run on the gpu (no memory check will be done).
 * --device <str>: Device for processing ("cpu","cuda") (optional, default: cuda)
+* --hires: Switch on hires processing (no conforming to 1mm, but to smallest voxel size).
+* --batch_size: Batch size for inference. Default=8
 
 
 ### Example Command Evaluation Single Subject
 To run the network on MRI-volumes of subject1 in ./data (specified by --i_dir flag; e.g. ./data/subject1/orig.mgz), change into the *FastSurferCNN* directory and run the following commands: 
 
 ```
-python3 eval.py --i_dir ../data \
---o_dir ../data \
+python3 eval.py --in_dir ../data \
+--sd ../data \
 --t subject1 \
---log temp_Competitive.log \
---network_sagittal_path ../checkpoints/Sagittal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
---network_coronal_path ../checkpoints/Coronal_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl \
---network_axial_path ../checkpoints/Axial_Weights_FastSurferCNN/ckpts/Epoch_30_training_state.pkl 
+--sg_log temp_Competitive.log \
```

Comment:
--seg_log 
 
Example should include a full path (or it should be mentioned below that this is possible).